commit b91f8e903b1004a35dfe01204852c628367475b7
Author: James Cloos <cloos@jhcloos.com>
Date:   Wed Aug 20 10:32:51 2008 -0400

    xaw8 is gone, use xaw7

diff --git a/configure.ac b/configure.ac
index d310de8..cc8cbfb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -37,7 +37,7 @@ AC_PROG_INSTALL
 AC_CHECK_LIB([kstat], [kstat_open])
 
 # Checks for pkg-config packages
-XAW_CHECK_XPRINT_SUPPORT(XLOAD)
+PKG_CHECK_MODULES(XLOAD, xaw7)
 
 AC_SUBST(XLOAD_CFLAGS)
 AC_SUBST(XLOAD_LIBS)
